Makefile

```diff
@@ -72,7 +72,7 @@ lint: licensecheck go-lint oas-lint
 #? crd: Generates CRD using controller-gen
 .PHONY: crd
 crd: controller-gen
-	${CONTROLLER_GEN} crd:crdVersions=v1 paths="./endpoint/..." output:crd:stdout > docs/contributing/crd-source/crd-manifest.yaml
+	${CONTROLLER_GEN} crd:crdVersions=v1 paths="./endpoint/..." output:crd:stdout > docs/sources/crd/crd-manifest.yaml
```

Comment:
this is not a location where manifest live currently. This is where it lives `charts/external-dns/crds/dnsendpoint.yaml` So location and name are different

Reply:
:thinking: It seems there are two places: 
1. https://github.com/kubernetes-sigs/external-dns/blob/master/docs/sources/crd/crd-manifest.yaml
2. https://github.com/kubernetes-sigs/external-dns/blob/master/charts/external-dns/crds/dnsendpoint.yaml

I'll see what I can do to either avoid duplicates or at least ensures that it's sync'd

Reply:
It should be synced, otherwise we end-up in up-to-date CRD and helm chart with some other CRD ;-)

This could be a follow-up pull request to sort out packaing, I do not mind

Reply:
Example external-secrets managing CRDs in helm chart https://github.com/external-secrets/external-secrets/tree/main/deploy/charts/external-secrets/templates/crds

But this could end-up in a holy war what is the right way manaing CRDs......